docs/billing.md

```diff
@@ -10,139 +10,171 @@ stack:         all
 
 
-### The monthly billing cycle in detail
+#### The monthly billing cycle in detail
 
-At the end of the month, the fortrabbit billing bot will go through all the Billing Contacts and check for the currents month usage. Based on that data, the monthly invoices will be created and made accessible from the Dashboard. The invoice is dated on the last day of the month and the service period is that month. On the first of the (next) month, the fortrabbit mail bot will send an invoice notice by e-mail to all clients. On the 3rd of the month, the fortrabbit billing bot will attempt to charge for all invoice with payment method credit card. Around the 7th of the month the SEPA direct payments for the invoices, for clients with that payment method will be issued and a few days later executed.
+The fortrabbit billing bot calculates the due amount for each Billing Contact at the end of the month. Based on that data, a new invoice will be created. The invoices are accessible through our Dashboard. The invoice is dated to the last day of the month and the service period is that month.
 
-At the middle of the month, a human will look after all the bounced invoices. That human also will attempt to issue payment again and will also might write addition payment reminders.
+* On the first day of the following month, all clients are notified about this new invoice.
+* On the 3rd of the month, the fortrabbit billing bot will charge all invoices having credit card as the payment method.
+* Around the 7th of the month, all the SEPA direct payments invoices are issued and executed a few days later.
+
+In the middle of a month, a human will look over bounced invoices. At that time, due payments will be re-charged and additional payment reminders may be sent out.
 
 
 ### Costs monitoring
 
-You can check the current costs with your Billing Contact, see [here](/billing-contact#toc-costs-monitoring).
+You can check the current costs by looking at your *Billing Contact*, see link:/billing-contact#toc-costs-monitoring[here].
 
 
 ### Payment methods
 
-You can pay by **credit card** (Visa, Master, Amex, no debit cards) or **SEPA direct debit** (EU clients only). For "enterprise clients" (large volume) we also offer to pay by **bill** (on account, after invoice).
+You can pay by credit card or SEPA direct debit. For enterprise clients (large volume) we also offer to pay by bill (giro / wire) (on account, after invoice).
+
+* We support credit cards including Visa, MasterCart and Amex.
+* SEPA direct debit is accessible to clients in the European Union.
+* We do not support debit cards.
+* We do not support payments via PayPal at this time and there are no future plans to integrate with PayPal.
```

Comment:
Suggestion: Use less "We". 

* Credit cards … are supported
* Debit cards are not ...

Reply:
ok.